session/session.go

```diff
@@ -953,6 +953,8 @@ func (s *session) execute(ctx context.Context, sql string) (recordSets []sqlexec
 		}
 		metrics.SessionExecuteCompileDuration.WithLabelValues(label).Observe(time.Since(startTS).Seconds())
 
+		s.sessionVars.StmtCtx.Normalized, s.sessionVars.StmtCtx.SQLDigest = parser.NormalizeDigest(stmtNode.Text())
```

Comment:
how about lazily compute the SQL digest when we need it?

Reply:
good idea, done, PTAL thx~